software/shelf.yml

```
@@ -0,0 +1,11 @@
+name: "Shelf"
+website_url: "https://www.shelf.nu"
+source_code_url: "https://github.com/Shelf-nu/shelf.nu"
+description: "Asset and equipment tracking software used by teams who value clarity. Shelf is an asset database and QR asset label generator that lets you create, manage and overview your assets across locations. Unlimited assets, free forever."
+licenses:
+  - AGPL-3.0
+platforms:
+  - Nodejs
+tags:
+  - Automation
+  - Inventory Management
```

```suggestion
  - Inventory Management
  - Maps and Global Positioning System (GPS)
```

I think `Inventory Management` alone is sufficient - the [Automation](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/tags/automation.yml) tag is used for generic/workflow automation tools that do not fit in any other category, but Shelf is clearly geared towards Inventory Management/asset tracking.

Many software projects *do* automate manual processes, so we should be careful when adding software to this category, or it will grow very large.

Since the software has location tracking features, you may want to add the [Maps and Global Positioning System (GPS)
](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/tags/maps-and-global-positioning-system-gps.yml) tag.